src/main/jbake/content/docs/components/btm/serverinstall.adoc

```diff
@@ -31,14 +31,17 @@ Therefore when starting the BTM server we will need to specify a port offset. Th
 
 [source,shell]
 ----
-bin/standalone.sh -Djboss.socket.binding.port-offset=100
+bin/standalone.sh -Djboss.socket.binding.port-offset=100 -Dkeycloak.server.url=http://localhost:8180/auth
 ----
 
-NOTE: When the server is initially started, it is not configured with any users. The BTM UI login page includes a link for registering a user. However, a default user profile is included within the distribution that includes a single user *_jdoe_* with password *_password_*. To import this default profile, the first time the server is started it should include the following _-Dkeycloak.import_ parameter:
+NOTE: If a port offset is used, then it is also necessary to tell _keycloak_ where to find the
+authorisation/authentication server.
```

Comment:
Would it make sense to add something like "in the above example the base port is 8080 and with an offset of 100, the KC port is now 8080+100=8180" ?